docs/gaps/teams-proposal/README.md

```diff
@@ -0,0 +1,188 @@
+# Supporting Teams and Hats in gittuf
+
+## Metadata
+
+* **Number:** TBD
+* **Title:** Supporting Teams and Hats in gittuf
+* **Implemented:** No
+* **Withdrawn/Rejected:** No
+* **Sponsors:** Yongjae Chung (yongjae354)
+* **Contributors:** Patrick Zielinski (patzielinski), Isabel Ayres (i-ayres),
+  Peter Ye (PeterJYE)
+* **Related GAPs:** [GAP-5](/docs/gaps/5/README.md)
+* **Last Modified:** October 16, 2025
+
+## Abstract
+Currently gittuf doesn't have a convenient way to group multiple identities
+together. In this GAP, we aim to increase the usability and scalability of
+gittuf by introducing the idea of teams, where multiple identities can be
+grouped together to be given equal privileges.
+
+## Motivation
+In certain scenarios, repository administrators may need to delegate
+authorization to a group of people. This group may be a team of developers,
+release managers, lawyers, QA, security, or etc., depending on the delegated
+namespace. [GAP-5](/docs/gaps/5/README.md) introduces the notion of a principal,
+which allows multiple keys to be grouped to a single entity. However, the
+current implementation of principals does not support grouping multiple entities
+into a single entity. This makes it difficult for a delegating entity to
+identify groups within a delegation (since they are implicit), as well as to
+modify all associated delegations when someone needs to be added to or removed
+from a group. With teams in gittuf, the delegating entity only needs to run an
+update to their team definition once, which will affect all delegations where
+the team is authorized.
+
+This GAP introduces the idea of a team in gittuf, in which multiple identities
+can be grouped together as part of a single team, and a single identity can
+be associated with multiple teams. 
+
+In addition to the definition of teams, the idea of wearing different "hats"
+must be supported. In some cases, people may be assigned to multiple teams, and
+must be able to select which team to represent when signing off a change using
+gittuf. For instance, a repository contributer Alice is a in 2 different teams:
```

Comment:
Is a contributor a developer?  How are they different?

(in other words, pick one technical term and repeat it.  Synonyms are really bad for technical documents.